README.md

```diff
@@ -1,52 +1,83 @@
-# Trans-Terminal Copy/Paste
+# ttcopy: Trans-Terminal Copy/Paste
+
+[![Build Status](https://travis-ci.org/greymd/ttcopy.svg?branch=master)](https://travis-ci.org/greymd/ttcopy)
+
 Provide copying and pasting within multiple hosts through the Web.
 
+![Introduction Image](./img/ttcp_intro_img.png)
+
 # Environment
-  * zsh
-  * bash
+  * zsh (tested ver: 4.3, 5.0)
+  * bash (tested ver: 3.2, 4.2)
 
-# Dependencies
+## Dependent commands
   * openssl
   * curl
 
 # Install
+First of all, please prepare ID and Password **as you like**.
+Do not be lazy. You are **NOT** required to register them on any services.
+Same data source (like clipboard) is shared within the hosts having same ID/Password.
```

Comment:
I feel "Same data source" is a bit vague. How about "The data you copied can be pasted within the hosts having same ID/Password"